include/rang.hpp

```diff
@@ -347,30 +471,37 @@ template <typename T>
 inline rang_implementation::enableStd<T> operator<<(std::ostream &os,
                                                     const T value)
 {
-    const int option = rang_implementation::controlValue();
+    const control option = rang_implementation::controlMode();
     switch (option) {
-        case 0: return os;
-        case 1:
+        case control::offColor: return os;
+        case control::autoColor:
             return rang_implementation::supportsColor()
                 && rang_implementation::isTerminal(os.rdbuf())
               ? rang_implementation::setColor(os, value)
               : os;
-        case 2: return rang_implementation::setColor(os, value);
+        case control::forceColor: return rang_implementation::setColor(os, value);
         default: return os;
     }
 }
 
+inline void setWinTermMode(const rang::winTerm value) noexcept
+{
+    rang_implementation::winTermMode() = value;
+}
+
+inline void setControlMode(const control value) noexcept
+{
+    rang_implementation::controlMode() = value;
+}
+
+// This function should be deprecated, cause it affects all streams but not only "os" stream.
+// Use rang::setControlMode instead.
 inline std::ostream &operator<<(std::ostream &os, const rang::control value)
```

Comment:
we can remove this entirely because the next release would be `3.0` which can take breaking changes 😄